samples/data-sealing/enclave_a_v1/CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+if(WIN32)
+  include(maybe_build_using_clangw)
+endif()
+
 add_executable(enclave_a_v1 ecalls.cpp)
```

Comment:
Oh you mean for the maybe_build_using_clangw having an if(WIN32) inside of it?  That's because we won't be distributing the maybe_build_using_clangw file with the Linux builds, so we can't invoke that function, even if it is a no-op on Linux!